packages/visual-editor/src/docs/local-editor.md

```
@@ -0,0 +1,139 @@
+for each Entity and Directory layout. Pages uses these templates to create
+local snapshots. The plug-in removes them when development stops and before
+production builds.
+
+## Test Data
+
+### ENTITY Page Sets
+
+By default, a basic location entity stream is used to pull data for ENTITY page set types.
+This can be configured in the `stream.config.ts` to use other entity types and to include other fields (including custom fields ).
+
+##### Alternative layouts
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h4; Actual: h5

(MD001, heading-increment)